_includes/homepage_content.html

```diff
@@ -30,8 +30,8 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
                               <div class="id-mac hidden">
                                   <div class="float-left">
-                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g">
-                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w">
+                                      <img src="/images/icon-apple-g.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
+                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-g" alt="Download Bisq for Mac">
```

Comment:
```suggestion
                                      <img src="/images/icon-apple-w.svg" class="os-icon os-icon-w" alt="Download Bisq for Mac">
```

Why was this changed?